### PR TITLE
fix(curriculum): added the missing backticks

### LINKED ISSUE #52789: Missing backticks - Step 49 of the JS (Beta) Certification - Learn Basic JavaScript by Building a Role Playing Game
### Describe the Issue

Hi,
I noticed that the word `text` is not surrounded by backticks (reference: step 49 of the JS Certification (Beta) - Learn Basic JavaScript by Building a Role Playing Game). The word with the missing backticks is in the last line of the instructions 

These are the current instructions:

"In your `goTown` function, change your `button` elements' `innerText` properties to be `Go to store`, `Go to cave`, and `Fight dragon`. Update your `onclick` properties to be `goStore`, `goCave`, and `fightDragon`, respectively.

Finally, update your text's `innerText` property to be `You are in the town square. You see a sign that says Store.`."

### Affected Page

https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures-v8/learn-basic-javascript-by-building-a-role-playing-game/step-49

### Your code

```
function goTown() {
  button1.innerText = "Buy 10 health (10 gold)";
  button2.innerText = "Buy weapon (30 gold)";
  button3.innerText = "Go to town square";
  button1.onclick = buyHealth;
  button2.onclick = buyWeapon;
  button3.onclick = goTown;
  text.innerText = "You enter the store.";
}
```

### Expected behavior

My suggestion would be to make the word "text" (located in the last line of the instructions) surrounded by backticks so it is clearer that this text refers to `div#text`

### Screenshots

step49fcc

### System

- Device: [e.g. iPhone 6, Laptop]
- OS: [e.g. iOS 14, Windows 10, Ubuntu 20.04]
- Browser: [e.g. Chrome, Safari]
- Version: [e.g. 22]

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by ShekharSahu404**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #52789

I added the missing backticks and there was another mistake that was elements' so i changed it to element's.

### CODE REVIEW

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a7c071219da921758a35bb.md [hunk 1]
```diff
@@ -7,9 +7,9 @@ dashedName: step-49
 
 # --description--
 
-In your `goTown` function, change your `button` elements' `innerText` properties to be `Go to store`, `Go to cave`, and `Fight dragon`. Update your `onclick` properties to be `goStore`, `goCave`, and `fightDragon`, respectively.
+In your `goTown` function, change your `button` element's `innerText` properties to be `Go to store`, `Go to cave`, and `Fight dragon`. Update your `onclick` properties to be `goStore`, `goCave`, and `fightDragon`, respectively.
 
-Finally, update your text's `innerText` property to be `You are in the town square. You see a sign that says Store.`.
+Finally, update your `text` `innerText` property to be `You are in the town square. You see a sign that says Store.`.
```
[the remark this answers is not in the document]
Comment (gikf) [reply curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a7c071219da921758a35bb.md:12]: I'd try to reword this, so there's no need for the _'s_, something like:
```suggestion
Finally, update `innerText` property of your `text` to be `You are in the town square. You see a sign that says Store.`.
```

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a7c071219da921758a35bb.md [hunk 2]
```diff
@@ -7,9 +7,9 @@ dashedName: step-49
 
 # --description--
 
-In your `goTown` function, change your `button` elements' `innerText` properties to be `Go to store`, `Go to cave`, and `Fight dragon`. Update your `onclick` properties to be `goStore`, `goCave`, and `fightDragon`, respectively.
+In your `goTown` function, change your `button` element's `innerText` properties to be `Go to store`, `Go to cave`, and `Fight dragon`. Update your `onclick` properties to be `goStore`, `goCave`, and `fightDragon`, respectively.
```
[the remark this answers is not in the document]
Comment (ShekharSahu404) [reply curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a7c071219da921758a35bb.md:10]: Thanks for the review and letting me know. I thought it apostrophe was misplaced, I should have confirmed it first.

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a7c071219da921758a35bb.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (ShekharSahu404) [reply curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a7c071219da921758a35bb.md:12]: I see, I will try to do the changes as per your suggestion.

**Comment by ShekharSahu404** [comment]
I have made changes as per your suggestion, please review it.
Thank you.

**Review by gikf — APPROVED** [review]
LGTM :+1:

**Review by huyenltnguyen — APPROVED** [review]
LGTM 👍

@ShekharSahu404 Thank you for the PR and congratulations on your first contribution to this repo 🎉
We look forward to your next.

Happy contributing!

**Pull request merged**